.github/workflows/code.yml

```diff
@@ -49,7 +50,12 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --target wasm32-unknown-unknown -p matchbox_socket -p bevy_ggrs_example -p simple_example
+          args: >
+            --all-targets
+            --target wasm32-unknown-unknown
+            -p matchbox_socket
+            -p bevy_ggrs_example
+            -p simple_example
```

Comment:
Oh, we're missing `bevy_matchbox` here. and probably in a few other places as well?

Reply:
Resolved in #172